src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
This is awesome, but the notifications screen isn't the only place to navigate to the issue screen. There are two other places in the app (`IssueListItem` component and in the `EventsScreen`). Could you update all the places in the app to have this state param? (you can most probably check for `issue.pull_request` like I do in other places of the app to identify whether an issue is actually a pull request)